src/icp_canister_backend/tests/test.rs

```diff
@@ -1,10 +1,15 @@
-use candid::{decode_one, encode_args, Principal, Nat};
+use candid::{decode_one, encode_args, Nat, Principal};
+use icp_canister_backend::TransferArg;
+use icp_canister_backend::{Account, DepositError};
 use pocket_ic::PocketIc;
 use sha2::{Digest, Sha256};
-use icp_canister_backend::{Account, DepositError};
 mod types;
 use types::*;
 
+lazy_static::lazy_static! {
+    static ref FEE: Nat = Nat::from(10_000u64);
```

Comment:
You have the same constant in canister named TRANSFER_FEE, keep the same/similar namigs for the same things in the code.
Name it TRANSFER_FEE too.